operator/Makefile

```diff
@@ -38,7 +38,7 @@ help: Makefile
 # Requires operator-sdk - download it from https://github.com/operator-framework/operator-sdk/releases
 operator-build: .ensure-operator-sdk-exists
 	@echo Build operator
-	${OP_SDK} build ${OPERATOR_IMAGE_NAME}:${OPERATOR_IMAGE_VERSION}
+	${OP_SDK} build "${OPERATOR_IMAGE_NAME}:${OPERATOR_IMAGE_VERSION}"
```

Comment:
you quoted the image name/version here, but not a few lines down (line 46) in docker push command - should these be quoted consistently?

Reply:
I suppose.